### ISSUE TITLE
Need to add --login to bash to work with rbenv

### ISSUE THREAD

**Issue by Gazer** [gojuno/mainframer#143]
Hi, I was trying maiframer but I was unable to run fastlane on the remote machine :

```
$ ./mainframer.sh fastlane help
:: mainframer v2.0.0

Sync local → remote machine...
Sync done: took 1 seconds.

Executing command on remote machine…

fastlane help

bash: line 1: fastlane: command not found

Execution failed: took 0 seconds.

Sync remote → local machine…
Sync done: took 6 seconds.

Failure: took 7 seconds.
```

After reading a lot, triying to use .bash_profile or .profile to setup rbenv correctly, I discovered that if I change the line 102 (of 2.0.0 release) to use `bash --login` it works as expected :

```
$ ./mainframer.sh fastlane help
:: mainframer v2.0.0

Sync local → remote machine...
Sync done: took 3 seconds.

Executing command on remote machine…

fastlane help

  fastlane

  CLI for 'fastlane' - The easiest way to automate beta deployments and releases for your iOS and Android apps
....
```

The .bashrc of the user I use to connecto to the remote machine ends with :

```
export PATH="$HOME/.rbenv/bin:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools:$PATH"
eval "$(rbenv init -)"
```

As I can see from the recipe to configure the remote machine there is nothing special to setup, but my ENV is different if I use or not --login. The remote system is a Ubuntu 14.04.4 LTS.

Please let me know if you need more info or some test from my side.

**Comment by artem-zinnatullin** [comment]
>>The .bashrc of the user I use to connecto to the remote machine ends with 

>ends with 

That's the reason, on most Linux distributives `~/.bashrc` has ~this in the beginning:

```bash
# ~/.bashrc: executed by bash(1) for non-login shells.
# see /usr/share/doc/bash/examples/startup-files (in the package bash-doc)
# for examples

# If not running interactively, don't do anything
case $- in
    *i*) ;;
      *) return;;
esac
```

Notice `# If not running interactively, don't do anything` which basically stops evaluation of following content of the `~/.bashrc`.

For example in our setup we put required environment variables **before** this line, simply in the beginning of `~/.bashrc` so they're evaluated no matter if it's login session or not.

**Comment by artem-zinnatullin** [comment]
So, I don't think we *need* to use `bash --login` but instead **highlight** this common issue in the [SETUP_REMOTE](https://github.com/gojuno/mainframer/blob/development/docs/SETUP_REMOTE.md) docs, @Gazer what do you think?

cc @ming13

**Comment by Gazer** [comment]
That makes sense, I've miss that detail of the bashrc. Moving the config to the top of the file solve the problem.

I think that leaving a note on the docs is enough. Many instruction (like rbenv) tell you to add the commands at the end of the bashrc and may save someone else in the future :).

Thanks you!

**Comment by artem-zinnatullin** [comment]
Ok! Wanna do a PR for docs 😺 or leaving it for us?

**Comment by arturdryomov** [comment]
Huh, looks strange actually, never had this in Zsh setup. Yep, I guess we can highlight that.

**Comment by Gazer** [comment]
I'll do the PR tonight!, right now I've to deliver some work 📦

**Comment by artem-zinnatullin** [comment]
@ming13 that's because I setup all the users we have and this was an issue I spent few hours on fixing…

@Gazer nice, will be happy to review then :)

**Comment by artem-zinnatullin** [comment]
Closed by #145!
